### ISSUE TITLE
Isuue when running esbmc wrapper

### ISSUE THREAD

**Issue by fatimahkj** [esbmc/esbmc#158]
Hi, 
I'm trying to run esbmc wrapper (I download it from here: https://gitlab.com/sosy-lab/test-comp/archives-2020/blob/master/2020/esbmc-falsi.zip)
and I used the simple C code in this paper
"ESBMC 6:1: Automated Test Case Generation using Bounded Model Checking"
The code : 

int main (){
unsigned int M = __VERIFIER_nondetuint();
int A[M], B[M], C[M];
unsigned int i;
for(i=0; i<M; i++) A[i] = __VERIFIER_nondetint();
for(i=0; i<M; i++) B[i] = __VERIFIER_nondetint();
for(i=0; i<M; i++) C[i]=A[i]+B[i];
for(i=0; i<M; i++) __VERIFIER_assert(C[i]==A[i]-B[i]);

}

The command I used is ./esbmc-wrapper.py -a 32 -p ./property_file/overflow -s falsi  smallcode.c 
 
However, I got this warning and there is no test cases generated.
![Screenshot from 2020-04-23 14-44-13](https://user-images.githubusercontent.com/47563480/80309399-b94b0100-87cc-11ea-9014-252c276b77eb.png)

Do you know how to fix the problem and generate the test cases?

Thank you for your help in advance.

**Comment by rafaelsamenezes** [comment]
Hello @fatimahkj, there is one issue:

- Non-deterministic should be in the format: `__VERIFIER_nondet_X()` so just add the missing `_` .

Also, I noticed that you are searching for overflows, which means the property: `CHECK( init(main()), LTL(G ! overflow) )`. When we developed the wrapper we assumed that only `COVER( init(main()), FQL(COVER EDGES(@CALL(__VERIFIER_error))) )` would generate testcases, which included some extra flags. 

I embeded a Zip containing the test-suite generated for your program (for overflows) and a custom wrapper. Let us know how it went!
[files.tar.gz](https://github.com/esbmc/esbmc/files/4535488/files.tar.gz)
